frontend/vue.config.js

```diff
@@ -1,6 +1,8 @@
 const path = require('path')
 const dotenv = require('dotenv-webpack')
 
+process.env.VUE_APP_BUILD_COMMIT = process.env.BUILD_COMMIT || '000000000'
```

Comment:
Why do you rename this? This makes no sense to me? Here yous set an default, but in the code above `process.env.VUE_APP_BUILD_COMMIT !== undefined` you check again?

Please use something like `CONFIG.BUILD_COMMIT` and define within the config your defaults - so the application only accesses cleansed data.

Reply:
I have first used CONFIG.BUILD_COMMIT but it didn't work. Only for environment variables starting with VUE_APP_ the content of the env variables will be copied over into the builded js. You know the frontend is running in the user browser, so it hasn't any access to the server environment variables. 

I've added the check because of the tests. Without this check many tests are failing.

Reply:
Ah ok, I see - you have to do that because of the clientside thing. maybe you csn find a way to properly ecapsulate it in the config.js - btw the other config options are also avsilable in the client, since the graphql endpoint is required by the client to make a connection.